Xamarin.Essentials/Platform/Platform.android.cs

```diff
@@ -18,8 +20,40 @@ public static partial class Platform
     {
         static ActivityLifecycleContextListener lifecycleListener;
 
-        internal static Context AppContext =>
-            Application.Context;
+        public static Context AppContext => Application.Context;
+
+        public static Activity Activity => lifecycleListener?.Activity;
```

Comment:
Maybe this should be named a bit differently to indicate the current/foreground activity.